metaflow/plugins/cards/card_decorator.py

```diff
@@ -55,11 +75,14 @@ class CardDecorator(StepDecorator):
             The or one of the cards attached to this step.
     """
 
+    _GLOBAL_CARD_INFO = {}
+
     name = "card"
     defaults = {
         "type": "default",
         "options": {},
         "scope": "task",
+        "rank": None,  # Can be one of "high", "medium", "low". Can help derive ordering on the UI.
```

Comment:
is the comment here correct? my expectation for "rank" would've been integer values. There's also no validation for the value entered.